src/ape_pm/project.py

```diff
@@ -266,6 +266,9 @@ def _parse_solidity_config(
         if evm_version := data.get("evm_version"):
             sol_cfg["evm_version"] = evm_version
 
+        if via_ir := data.get("via_ir"):
+            sol_cfg["viaIR"] = via_ir
```

Comment:
It is snake case: https://github.com/ApeWorX/ape-solidity/blob/main/ape_solidity/compiler.py#L142

```suggestion
            sol_cfg["via_ir"] = via_ir
```